doc/ExampleImplementations.xml

```diff
@@ -190,7 +190,7 @@ true
     </Section>
 
     <Section Label="Example_VertexCounter"><Heading>VertexCounter</Heading>
-        The method <K>VertexCounter</K> (<Ref Subsect="VertexCounter"/>) 
+        The method <K>VertexCounter</K> (<Ref Subsect="CounterOfVertices"/>) 
```

Comment:
Is VertexCounter an alias for CounterOfVertices?

Reply:
I mean, that the function VertexCounter also exists and does the same as CounterOfVertices. Is this the case?